dts/riscv/riscv32-litex-vexriscv.dtsi

```diff
@@ -34,8 +34,9 @@
 		compatible = "litex,vexriscv";
 		ranges;
 		intc0: interrupt-controller@bc0 {
-			#interrupt-cells = <2>;
 			compatible = "vexriscv-intc0";
+			#address-cells = <0>;
```

Comment:
Why is `address-cells` set to `0` here (and in other places, where `reg` is also defined?